tests/message.cpp

```diff
@@ -179,6 +179,45 @@ TEST_CASE("message equality non equal lhs empty", "[message]")
     CHECK(msg_a != msg_b);
 }
 
+TEST_CASE("message rebuild with size", "[message]")
+{
+    zmq::message_t msg();
```

Comment:
You need to remove the () because of most vexing parse. Same for all default constructors below